README.md

```diff
@@ -1074,6 +1074,7 @@ _Related: [Groupware](#groupware)_
 
 _Related: [Note-taking & Editors](#note-taking--editors), [Wikis](#wikis)_
 
+- [Atomic Server](https://github.com/atomicdata-dev/atomic-data-rust) - Knowledge graph database with documents (similar to Notion), tables, search, and a powerful linked data API. Lightweight, very fast and no runtime dependencies. ([Demo](https://atomicdata.dev/)) `MIT` `Docker/Rust`
```

Comment:
> https://github.com/atomicdata-dev/atomic-data-rust/blob/master/server/README.md#installation--getting-started

> You can run atomic-server in four ways:
> 
> From a published [binary](https://github.com/joepio/atomic/releases) (probably the quickest)
> Using docker
> Install a desktop build (macOS only as of now)
> Using [Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) from crates.io: cargo install atomic-server
>     Manually from source

The recommended installation method is from precompiled single binary, please change the language/platform tag to `Rust/Docker` to reflect this.

Reply:
Instead of changing the PR, I've updated the README of Atomic-Server to suggest Docker images first. I think that's easier for most users, anyway!